opendata.swiss/ui/app/components/dataset/OdsMultiSelect.vue

```diff
@@ -19,39 +19,29 @@
         :options="options"
         :name="name"
         label="title"
-        :reduce="(option: Item) => option.id"
-
-        :selectable="
-          (option: Item) =>
-            !excluded.includes(option.id) &&
-            (selectLimit
-              ? !currentSelected || currentSelected.length < selectLimit
-              : true)
-        "
-        @update:model-value="emitNewValues"
+        :filterable="!!loadOptions"
+        :close-on-select="closeOnSelect"
+        @search="loadOptions!"
       >
+        <template #no-options>
+          <slot name="no-options"/>
+        </template>
         <!-- Workaround for required validation -->
         <template #search="{ attributes, events }">
           <input
             class="vs__search"
             :required="
-              required && (!currentSelected || currentSelected.length === 0)
+              required && (!selected || selected.length === 0)
             "
             v-bind="attributes as any"
             v-on="events"
           >
         </template>
-       <template #option="option">
-        <span>
-          {{ option.title }}
-          <span v-if="option.count" style="float: right; color: #888;">({{ option.count }})</span>
-        </span>
-      </template>
-        <template
-          #selected-option="option"
-        >
-          {{ option.title }}
-          <input type="hidden" :name="name" :value="option.id">
+        <template #option="option">
+          <slot name="option" v-bind="option" />
+        </template>
+        <template #selected-option="option">
+          <slot name="selected-option" v-bind="option" />
```

Comment:
I moved the slots to the usage of multi-select so that it can be customised between the use cases of dataset filters, and the two applications in showcase form (categories and datasets)